lib/common_test/src/ct_snmp.erl

```diff
@@ -293,19 +297,18 @@ set_values(Agent, VarsAndVals, MgrAgentConfName, Config) ->
     end,
     SnmpSetReply.
 
-%%% @spec set_info(Config) -> [{Agent, OldVarsAndVals, NewVarsAndVals}] 
-%%%
-%%%      Config = [{Key, Value}] 
-%%%	 Agent = agent_name()
-%%%      OldVarsAndVals = varsandvals()
-%%%      NewVarsAndVals = varsandvals()
-%%%
 %%% @doc Returns a list of all successful set requests performed in
 %%% the test case in reverse order. The list contains the involved
 %%% user and agent, the value prior to the set and the new value. This
 %%% is intended to facilitate the clean up in the end_per_testcase
 %%% function i.e. the undoing of the set requests and its possible
 %%% side-effects.
+%%% @end
+-spec set_info(Config) -> [{Agent, OldVarsAndVals, NewVarsAndVals}] when
+      Config :: agent_config(),
```

Comment:
This should not be agent_config(). It is the Config variable from the test case, defined as [{Key,Value}], where Key::atom(), Value::term().